Formula/fdroidcl.rb

```diff
@@ -0,0 +1,23 @@
+class Fdroidcl < Formula
+  desc "F-Droid desktop client"
+  homepage "https://github.com/mvdan/fdroidcl"
+  url "https://github.com/mvdan/fdroidcl/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "532a8c4c93216cbf13378ff409c06a08d48e8baee6119a50ed43dc0ce9ec7879"
+  license "BSD-3-Clause"
+  head "https://github.com/mvdan/fdroidcl.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Have you tried doing `brew livecheck fdroidcl` without this defined? The default one is usually good enough.

Reply:
Yeah, it's working just fine without this defined.